ios/FluentUI/Core/Theme/FluentColor.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import SwiftUI
+
+/// Wrapper view that returns the appropriate `Color` for a given `ColorSet` in a given environment.
+struct FluentColor: View {
+
+    /// Representative of the various color values that the rendered `Color` body could represent.
+    let colorSet: ColorSet
+
+    var body: some View {
+        color(from: colorValue)
+    }
+
+    // MARK: - Private
+
+    @Environment(\.colorSchemeContrast) private var contrast: ColorSchemeContrast
+    @Environment(\.colorScheme) private var colorScheme: ColorScheme
+
+    // TODO: How do we know if we're elevated?
```

Comment:
For UIKit: https://developer.apple.com/documentation/uikit/uiuserinterfacelevel/elevated

Not sure if SwiftUI supports anything like that.